client/tokenized-express-checkout/transformers/wc-to-stripe.js

```diff
@@ -96,7 +96,7 @@ export const transformCartDataForDisplayItems = ( cartData ) => {
 export const transformCartDataForShippingOptions = ( cartData ) =>
 	cartData.shipping_rates[ 0 ].shipping_rates.map( ( rate ) => ( {
 		id: rate.rate_id,
-		label: decodeEntities( rate.name ),
+		displayName: decodeEntities( rate.name ),
```

Comment:
Slight variation on how the ECE names each shipping rate, compared to PRB: https://docs.stripe.com/js/elements_object/express_checkout_element_shippingaddresschange_event#express_checkout_element_on_shipping_address_change-handler-resolve-shippingRates